transformer/dockerfilegenerator/nodejsdockerfiletransformer.go

```diff
@@ -39,6 +41,8 @@ const (
 	packageJSONFile    = "package.json"
 )
 
+var supportedNodeVersions = []int{14, 16, 12, 10}
```

Comment:
We should read this from a mapping file containing the version numbers and tag names, similar to what we do with java and dotnet.

Reply:
Done.